src/commands/android/utils/adbWirelessConnect.ts

```diff
@@ -0,0 +1,73 @@
+import {Platform} from '../interfaces';
+import {execBinarySync} from './sdk';
+import Logger from '../../../logger';
+import inquirer from 'inquirer';
+import colors from 'ansi-colors';
+
+export async function connectAdbWirelessly(adbLocation: string, platform: Platform): Promise<boolean> {
+  try {
+    const deviceList = execBinarySync(adbLocation, 'adb', platform, 'devices -l');
+    if (deviceList !== null) {
+      if (!deviceList.toString().includes('transport_id')) {
+        Logger.log(colors.red('No devices found. Please connect a device and try again.'));
+
+        return false;
+      }
+    }
+
+    Logger.log(`${colors.bold('Follow the below steps to connect to your device wirelessly:\n')}
+  1. Enable USB debugging on your device.
+  2. Connect your device to your computer using a USB cable.
+  3. Grant permission for File transfer on your device.
+  4. Connect your device to the same network as your computer ${colors.grey('(you may connect your device to your computer\'s hotspot)')}.
+  5. Enable wireless debugging on your device.
+  6. Find the IP address, port number and pairing code of your device by going to 
+     ${colors.cyan('Settings > Wireless Debugging > Pair device with pairing code.')}\n`);
```

Comment:
Pairing port number is different from the actual port number of the device for wireless debugging, we should distinguish them.
```suggestion
  6. Find the IP address, pairing port number and pairing code of your device by going to 
     ${colors.cyan('Settings > Wireless Debugging > Pair device with pairing code.')}\n`);
```